lib/routes/weread/namespace.ts

```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```

Comment:
Please [use existing namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) https://github.com/DIYgod/RSSHub/blob/cc55dbf85bf7e969a4e3d957d517690b2d82ee67/lib/routes/qq/namespace.ts#L3-L8 instead of creating variation.